ceph_medic/checks/osds.py

```diff
@@ -0,0 +1,31 @@
+
+
+#
+# Utilities
+#
+
+def get_osd_ceph_fsids(data):
+    fsids = []
+    for file_path in data['paths']['/var/lib/ceph']['files'].keys():
+        if "ceph_fsid" in file_path:
+            fsids.append(data['paths']['/var/lib/ceph']['files'][file_path]['contents'])
+    return set(fsids)
+
+
+def check_osd_ceph_fsid(host, data):
+    code = 'WOSD1'
+    msg = """
+    Multiple ceph_fsid values found in /var/lib/ceph/osd.
```

Comment:
The output looks nice enough with one warning, but I guess you're worried about if there are many warnings or errors? Even if it's one line printing the fsid values will take it to two at least.

```
(ceph-medic)Andrews-MacBook-Pro-2:jrn-col andrewschoen$ ceph-medic --ssh-config vagrant_ssh_config check
Host: mon0                  connection: [connected  ]
Host: osd0                  connection: [connected  ]
Collection completed!

=======================  Starting remote check session  ========================
Version: 1.0.0    Cluster Name: "test"
Total hosts: [2]
OSDs:    1    MONs:    1     Clients:    0
MDSs:    0    RGWs:    0     MGRs:       0

================================================================================

------------ osds ------------
 osd0
   WOSD1:
    Multiple ceph_fsid values found in /var/lib/ceph/osd.

    This might mean you are hosting OSDs for many clusters on
    this node or that some OSDs are misconfigured to join the
    clusters you expect.

    ceph_fsid values found:
    70789b91-1ece-4909-b179-d045467a35a7
    !!70789b91-1ece-4909-b179-d045467a35a7


------------ mons ------------
 mon0
```